### ISSUE TITLE
Unsatisfied npm version 6.5.0-next.0. Please update npm to >=3.0.0 before you continue

### ISSUE THREAD

**Issue by hckhanh** [adonisjs/adonis-cli#121]
## Node.js and npm version
Node: `11.6.0`
NPM: `6.5.0-next.0`

## Sample Code (to reproduce the issue)
Just run the cli with the npm version above and I received this

```bash
  [1/6] ❌  Unsupported npm version [node & npm]

   ERROR  Unsatisfied npm version 6.5.0-next.0. Please update npm to >=3.0.0 before you continue
```

**Comment by Yoeri-Wysselinck** [comment]
Same here :(

**Comment by Yoeri-Wysselinck** [comment]
Found it! You can't run the "Latest Version" of Nodejs. You neet the "LTS" version, then it'll work

**Comment by hckhanh** [comment]
@Yoeri-Wysselinck maybe the way npm name their package version is the problem. It did not follow the semver specification.

**Comment by RomainLanz** [comment]
Should be fixed, may you download the new version via Github and give it a try to confirm it's also fixed on your side?

**Comment by thetutlage** [comment]
Released as `4.0.11`. Just update the CLI
